.projenrc.ts

```diff
@@ -15,8 +17,6 @@ const project = new typescript.TypeScriptProject({
   autoApproveUpgrades: true,
 });
 
-project.package.addField('private', true);
```

Comment:
@mrgrain i believe that we default `private: true` in cdklabs projen types, which is why this line is simply deleted right

Reply:
Yes, that's a manual change a made